scripts/provisioner/update_solution.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+source "$WORKSPACE"/solutions/kubernetes/functions.sh
+
+HOST_FILE=$PWD/hosts
+SCRIPT_PATH=/root/cortx-k8s/k8_cortx_cloud
+YQ_VERSION=v4.13.3
+YQ_BINARY=yq_linux_386
+
+cp "$WORKSPACE"/scripts/provisioner/hosts "$WORKSPACE"/solutions/kubernetes/hosts
+
+function install_yq_module() {
+    pip3 show yq && pip3 uninstall yq -y
+    wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY}.tar.gz -O - | tar xz && mv ${YQ_BINARY} /usr/bin/yq
+    if [ -f /usr/local/bin/yq ]; then rm -rf /usr/local/bin/yq; fi
+    ln -s /usr/bin/yq /usr/local/bin/yq
+}
+
+function add_image_solution_config() {
+    pushd "$WORKSPACE"/scripts/provisioner
+        image=$CONTROL_IMAGE yq e -i '.solution.images.cortxcontrol = env(image)' solution_template.yaml
```

Comment:
Can't you use solution.yaml from `cortx-k8s` repo itself instead of keeping its copy in folder. drawback with this is that if any changes, you need to raise PR here again.
I guess you can use a var for solution.yaml file where you already have path variable `SCRIPT_PATH`. create new var and extend it to add `solution.yaml` in path.

Reply:
Problem was to copy solution.yaml which comes as part of cortx-k8s to jenkins node/server. So, for now we clone cortx-k8s repo in jenkins node too and then edit the solution.yaml with images, nodes and storage and then copy it to setup VMs.